### PR TITLE
MATH-1573 - Redundant local variable

### PR DESCRIPTION
**Pull request by arturobernalg**

### CODE REVIEW

File: src/main/java/org/apache/commons/math4/stat/descriptive/moment/Variance.java [hunk 1]
```diff
@@ -418,11 +418,10 @@ public double evaluate(final double[] values, final double mean,
                     accum += dev * dev;
                     accum2 += dev;
                 }
-                double len = length;
```
Comment (ebourg) [inline src/main/java/org/apache/commons/math4/stat/descriptive/moment/Variance.java:421]: `len` is a double but `length` is an int, I suspect it's preferable to cast only once here.

Comment (arturobernalg) [reply src/main/java/org/apache/commons/math4/stat/descriptive/moment/Variance.java]: Hi @ebourg 
sounds fair. Changed
TY

File: src/main/java/org/apache/commons/math4/stat/inference/KolmogorovSmirnovTest.java [hunk 2]
```diff
@@ -163,14 +163,13 @@ public double kolmogorovSmirnovTest(ContinuousDistribution distribution, double[
     public double kolmogorovSmirnovStatistic(ContinuousDistribution distribution, double[] data) {
         checkArray(data);
         final int n = data.length;
-        final double nd = n;
```
Comment (ebourg) [inline src/main/java/org/apache/commons/math4/stat/inference/KolmogorovSmirnovTest.java:166]: Same observation for the casting here

File: src/main/java/org/apache/commons/math4/stat/descriptive/moment/Kurtosis.java [hunk 3]
```diff
@@ -183,12 +183,11 @@ public double evaluate(final double[] values, final int begin, final int length)
             accum3 /= FastMath.pow(stdDev, 4.0d);
 
             // Get N
-            double n0 = length;
```
Comment (ebourg) [inline src/main/java/org/apache/commons/math4/stat/descriptive/moment/Kurtosis.java:186]: Same here

File: src/main/java/org/apache/commons/math4/stat/descriptive/moment/Mean.java [hunk 4]
```diff
@@ -165,17 +165,16 @@ public double evaluate(final double[] values, final int begin, final int length)
 
         if (MathArrays.verifyValues(values, begin, length)) {
             Sum sum = new Sum();
-            double sampleSize = length;
```
Comment (ebourg) [inline src/main/java/org/apache/commons/math4/stat/descriptive/moment/Mean.java:168]: And here

File: src/main/java/org/apache/commons/math4/util/FastMath.java [hunk 5]
```diff
@@ -674,8 +674,6 @@ public static double tanh(double x) {
           double yb = -(ya - hiPrec[0] - hiPrec[1]);
 
           /* Numerator */
```
Comment (ebourg) [inline src/main/java/org/apache/commons/math4/util/FastMath.java:676]: This comment should go away too

Comment (arturobernalg) [reply src/main/java/org/apache/commons/math4/util/FastMath.java]: Changed

File: src/main/java/org/apache/commons/math4/stat/descriptive/moment/Mean.java (hunk 4 above)
Comment (arturobernalg) [reply src/main/java/org/apache/commons/math4/stat/descriptive/moment/Mean.java]: Changed

File: src/main/java/org/apache/commons/math4/stat/descriptive/moment/Kurtosis.java (hunk 3 above)
Comment (arturobernalg) [reply src/main/java/org/apache/commons/math4/stat/descriptive/moment/Kurtosis.java]: Changed

File: src/main/java/org/apache/commons/math4/stat/inference/KolmogorovSmirnovTest.java (hunk 2 above)
Comment (arturobernalg) [reply src/main/java/org/apache/commons/math4/stat/inference/KolmogorovSmirnovTest.java:166]: Changed

**Pull request closed without merging**